src/nvim/ex_cmds.c

```diff
@@ -5935,3 +6036,191 @@ void set_context_in_sign_cmd(expand_T *xp, char_u *arg)
     }
   }
 }
+
+/// Open a window for future displaying of the inc_sub mode.
+///
+/// Does not allow editing in the window.
+/// Returns when the window is closed.
```

Comment:
This sounds almost like "wait until the user [or some other code] closes the window, then return". Maybe "closes the window and restores the window layout before returning"